docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -38,6 +38,8 @@ At a high level, the setup steps are as follows:
 1. Follow the applicable instructions for the deployment option you chose, out of the following:
    - [Terraform Template](#deployterraform-template)
    - [gcloud CLI](#deploygcloud-cli)
+1. Enable egress rule if required.
+   - [Enable egress rule]
```

Comment:
This looks like half link formatting. Suggest:
- See [Confidential Space Account Setup](#confidential-space-account-setup) step 4.